README.md

```diff
@@ -95,6 +95,7 @@ This is the Stackdriver Metadata agent.
        $ brew install boost\@1.60 -c++11 && \
          (cd /usr/local/lib && ln -s libboost_thread-mt.a libboost_thread.a && \
           ln -s libboost_thread-mt.dylib libboost_thread.dylib)
+       $ brew link --force boost\@1.60
```

Comment:
This is part of installing boost, so should be in the previous command. How about
```sh
       $ brew install boost\@1.60 -c++11 && brew link --force boost\@1.60 && \
         (cd /usr/local/lib && ln -s libboost_thread-mt.a libboost_thread.a && \
          ln -s libboost_thread-mt.dylib libboost_thread.dylib)
```

Reply:
Good idea, fixed.